test/variable_rate_reactions.jl

```diff
@@ -1,48 +1,35 @@
 using DiffEqBiological, DiffEqJump, DiffEqBase, OrdinaryDiffEq, StochasticDiffEq
 using Base.Test
+using tmpMod
 
-r1 = VariableRateReaction(1e-4,(1,2),((1,-1),(2,1)))
-r2 = VariableRateReaction(0.01,[2],[(2,-1),(3,1)])
+sir_model = @reaction_network rn begin
+    0.1/1000, s + i --> 2i
+    0.01, i --> r
```

Comment:
Is this making use of variable rate reactions?

Reply:
No

Reply:
We'll need a way to set that if we want to do this kind of mixing then, or not support it.